admin-dev/themes/new-theme/js/pages/product/components/categories/categories-manager.ts

```diff
@@ -148,21 +148,24 @@ export default class CategoriesManager {
     }
 
     const defaultCategoryId = this.getDefaultCategoryId();
-    selectElement.innerHTML = '';
+
+    if (selectElement !== null && selectElement !== undefined) {
+      selectElement.innerHTML = '';
+    }
 
     categories.forEach((category) => {
       const optionElement = document.createElement('option');
       optionElement.value = String(category.id);
       optionElement.innerHTML = category.displayName;
       optionElement.selected = category.id === defaultCategoryId;
 
-      selectElement.append(optionElement);
+      selectElement?.append(optionElement);
     });
   }
 
   private listenDefaultCategorySelect(): void {
-    $(`#${this.defaultCategoryInput.id}`).on('change', (e) => {
-      const {currentTarget} = e;
+    this.defaultCategoryInput?.addEventListener('change', (e) => {
+      const currentTarget = e.currentTarget as HTMLInputElement;
```

Comment:
```suggestion
    this.defaultCategoryInput?.addEventListener('change', (e: {currentTarget: HTMLInputElement}) => {
      const {currentTarget} = e;
```

or something similar would be better